queue/amqp.go

```diff
@@ -72,27 +72,43 @@ func connect(url string) (*amqp.Connection, *amqp.Channel) {
 	// first try to connect again
 	var conn *amqp.Connection
 	var err error
+	logErr := true
 	for {
 		conn, err = amqp.Dial(url)
 		if err != nil {
-			log15.Error("error connecting to amqp", "err", err)
+			if logErr {
```

Comment:
maybe instead of just `bool` (was logged or not) we may have `var oldErr error` and inside `for` we may check if current error is equal previous one. In other words if we cannot connect but because of other reason we log it otherwise (for the same error) we can skip it - WDYT?

Reply:
ok, so just like in https://github.com/src-d/borges/pull/260 :+1: